common/modules/gcp-project/main.tf

```diff
@@ -23,7 +23,49 @@ variable "organization_id" {}
 
 variable "serviceaccount_key" {}
 
-variable "project_id" {} # id of the project which owns the credentials used by the provider
+# Id of the project which owns the credentials used by the provider
+variable "project_id" {}
+
+# List of APIs in use by the project without audit configuration
```

Comment:
Can you link to some Google documentation or something that explains how this list differs from the `_with_` list?

Reply:
Done.